website/content/docs/agent/autoauth/index.mdx

```diff
@@ -198,11 +198,49 @@ These configuration values are common to all Sinks:
 - `config` `(object: required)` - Configuration of the sink itself. See the
   sidebar for information about each sink.
 
-### Auto Auth Example
+### Auto Auth Examples
+
+Auto-Auth configuration objects take two separate forms when specified in `HCL`
+and `JSON`. The following examples are meant to clarify the differences between
+the two formats.
+
+#### HCL sinks
```

Comment:
Would it be better to rephrase this to `sinks in HCL format`?

Reply:
Good call, thanks!